{{cookiecutter.package_name}}/src/{{cookiecutter.namespace}}/{{cookiecutter.name}}/browser/configure.zcml

```diff
@@ -8,12 +8,12 @@
   <include package="z3c.jbot" file="meta.zcml" />
   <browser:jbot
       directory="overrides"
-      layer="{{cookiecutter.package_name}}.interfaces.I{{cookiecutter.project_slug.capitalize()}}CoreLayer"
+      layer="..interfaces.I{{cookiecutter.project_slug.capitalize()}}CoreLayer"
```

Comment:
No relative imports anywhere please! That's an anti-pattern.

Reply:
done